### PR TITLE
✨ Automatically create cnquery update PR

### PR DESCRIPTION
**Pull request by czunker**
A new cnquery release will trigger a workflow in the cnspec repo to create a PR which updates the dep.

### CODE REVIEW

File: .github/workflows/bump-cnspec.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        default: "v9.14.0"
+
+jobs:
+  trigger-cnspec:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger cnquery bump in cnspec
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.CNSPEC_REPO_API_TOKEN }}
```
Comment (benr) [inline .github/workflows/bump-cnspec.yml]: Use RELEASR_ACTION_TOKEN for the moment.

File: .github/workflows/bump-cnspec.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'cnquery version to use'
+        type: string
+        required: false
+        default: "v9.14.0"
```
Comment (chris-rock) [inline .github/workflows/bump-cnspec.yml]: Should this not be empty? I also feel that the workflow dispatch should be in cnspec not in cnquery

Comment (czunker) [reply .github/workflows/bump-cnspec.yml]: Good point.

We could use the `workflow_dispatch` to test the complete workflow, but then I'd need to rewrite this workflow. I'll remove it for now.

File: .github/workflows/bump-cnspec.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Trigger cnquery bump in cnspec
+
+on:
+  push:
+    tags:
+      - '*'
```
Comment (chris-rock) [inline .github/workflows/bump-cnspec.yml]: Ben just discovered that this should only happen on release. This way we would avoid that we have accidental pushes to other repos

Comment (czunker) [reply .github/workflows/bump-cnspec.yml]: Thanks. I changed this.

I'm not sure what you mean by "pushes to other repos". This workflow does not (git) push anything. It dispatches an event to the other repo and will there trigger a workflow.

**Review by benr — APPROVED** [review]

File: .github/workflows/bump-cnspec.yml (hunk 3 above)
Comment (benr) [reply .github/workflows/bump-cnspec.yml]: Thanks, looks great.

**Pull request merged**